types/base.d.ts

```diff
@@ -120,5 +120,13 @@ export interface ContextualCallbackTestContext extends CallbackTestContext {
 	context: any;
 }
 
+export interface Macro<T, I, E> {
+	(t: T, input: I, expected: E): void;
+	title? (providedTitle: string, input: I, expected: E): string;
```

Comment:
Last small remark. Should we call this `title: string` instead of `providedTitle: string`? Autocompletion tools show the param names and `title: string` might be better.

// @ivogabe @sindresorhus